### PR TITLE
Feature/add vggface2 labels

### PR DESCRIPTION
**Pull request by timesler**

### CODE REVIEW

**Comment by timesler** [comment]
Fixes #60

File: models/inception_resnet_v1.py [hunk 1]
```diff
@@ -300,6 +302,13 @@ def forward(self, x):
         else:
             x = F.normalize(x, p=2, dim=1)
         return x
+    
+    def load_labels(self, pretrained):
+        if pretrained is not 'vggface2':
+            return None
+        with open(os.path.join(os.path.dirname(__file__), f'../data/labels-vggface2.csv')) as f:
```
Comment (DavidKatz-il) [inline models/inception_resnet_v1.py:309]: `f-string` is supported only from `python>=3.6`, `Travis CI` also runs on python 3.5
So if you want to use `f-string` you need to drop `python 3.5`  from `CI`, and also updateing `setup.py`.

**Review by DavidKatz-il — CHANGES_REQUESTED** [review]
Thanks for this project.
I suggest dropping support for python3.5.

**Comment by j-adamczyk** [comment]
@timesler @DavidKatz-il are there any plans to add this? It would be really useful, since as-is, the Keras VGGFace implementation has this functionality, but facenet does not.
